### PR TITLE
Add --workdir flag to brew updater

### LINKED ISSUE #700: (super easy) add `--workdir` flag to `brew` updater
# Solution

We should explicitly set `--workdir ~/.local/opt/brew/`.

See:

https://github.com/webinstall/webi-installers/blob/4b8f62a1b805e588700d029ab44ddb530045d849/brew/brew-update-service-install#L9-L11

# Problem

```sh
brew-update-service-install
```

```text
OPTIONS: Making some assumptions...

	# Because this is your current working directory
	  --workdir /Users/aj/.local/opt/brew
````

### PR DESCRIPTION
**Pull request by Virtual4087**
Fixes #700

### CODE REVIEW

**Comment by coolaj86** [comment]
I will pull this in tomorrow.

File: brew/brew-update-service-install [hunk 1]
```diff
@@ -8,6 +8,7 @@ main() { (
 
     env PATH="$PATH" serviceman add --user \
         --name sh.brew.updater -- \
+        --workdir ~/.local/opt/brew/ \
```
Comment (coolaj86) [inline brew/brew-update-service-install:11]: this needs to move up one line

```diff
+        --workdir ~/.local/opt/brew/ \
          --name sh.brew.updater -- \
-        --workdir ~/.local/opt/brew/ \
```

Comment (coolaj86) [inline brew/brew-update-service-install:11]: this needs to move up one line

```diff
+        --workdir ~/.local/opt/brew/ \
          --name sh.brew.updater -- \
-        --workdir ~/.local/opt/brew/ \
```

The `--` means "don't accept flags after me"

**Review by coolaj86 — CHANGES_REQUESTED** [review]
this needs to move up one line

```diff
+        --workdir ~/.local/opt/brew/ \
          --name sh.brew.updater -- \
-        --workdir ~/.local/opt/brew/ \
```

**Comment by coolaj86** [comment]
I would have made that change myself, but you unchecked "allow maintainer commits" when making the PR.

**Comment by Virtual4087** [comment]
> I would have made that change myself, but you unchecked "allow maintainer commits" when making the PR.

my bad I didn't notice it, I'll make the changes right away

**Comment by coolaj86** [comment]
@Virtual4087 Now it's swapped around the other way. I updated my comment to hopefully be more clear.

**Comment by Virtual4087** [comment]
@coolaj86 can i open a new pr with a single commit and proper changes?

**Comment by coolaj86** [comment]
@Virtual4087 it's good to just learn how to use `git` rather than make a new PR.

```sh
# reset the branch back to the beginning
git reset --hard main
git pull --rebase https://github.com/webinstall/webi-installers main

# make your edits and commit
git commit -m "fix(brew): add --workdir to updater service"

# replace the feature branch
git push --force
```

**Comment by Virtual4087** [comment]
Ok I'm on it. Thanks for the explanation

**Review by Virtual4087 — COMMENTED** [review]
Thanks you @coolaj86. I learned something new today. This month has really taught me a lot about git.

**Review by coolaj86 — APPROVED** [review]
@Virtual4087 Thanks!

And I'm glad you were able to learn more about git. I wish more people would - because it's a great tool that solves so many problems that couldn't be solved ~15 years ago (before it went mainstream).

Also, apparently it was some bad settings on my end that were giving me an error that I assumed was you having unchecked the "allow maintainer edits" button.

**Pull request merged**